generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -20,108 +20,92 @@ package <%= packageName %>.web.rest;
 
 <%_
 
-/**
- * Returns a Java generator that randomizes a value.
- *
- * @param {string} type - Java type
- * @returns {string} java primary key value
- */
-function getJavaValueGeneratorForType(type) {
-  if (type === 'String') {
-    return 'UUID.randomUUID().toString()';
-  }
-  if (type === 'UUID') {
-    return 'UUID.randomUUID()';
-  }
-  if (type === 'Long') {
-    return 'count.incrementAndGet()';
-  }
-  throw new Error(`Java type ${type} does not have a random generator implemented`);
-}
-
```

Comment:
This function was implemented for 1 reason generate a unique id for manually generated ids and is applicable at this file only.
While `getPrimaryKeyValue` is to get a value value for tests that don't need uniqueness and is used for other tests.

`getPrimaryKeyValue(field.fieldType, databaseType, 'count.incrementAndGet()', true)` would make harder to add a unique value generator like a relationship since the fallback is fixed.

IMO this function should be kept.
While `getPrimaryKeyValue` could reuse `field.generateFakeData('java')` (java alternative is not implemented yet).